x/feesplit/spec/01_concepts.md

```diff
@@ -23,7 +23,7 @@ As described above, developers will earn a portion of the transaction fee after
 
```

Comment:
Nit: docs grammar for Fee Distribution (not in CL); 
The registration of factory contracts (smart contracts that have been deployed by other contracts) requires the identification **_of the_** original contract's deployer. This is done through address derivation.